app/components/finance/npq/payment_overviews/output_payment_row.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    module PaymentOverviews
+      class OutputPaymentRow < BaseComponent
+        include FinanceHelper
+
+        attr_accessor :output_payment, :contract, :statement
+
+        def initialize(output_payment, contract, statement)
+          @output_payment = output_payment
+          @contract = contract
+          @statement = statement
+        end
+
+        def total
+          output_payment[:subtotal]
+        end
+
+        def payment_per_trainee
+          output_payment[:per_participant]
+        end
+
+        def current_trainees
+          statement
+            .participant_declarations
```

Comment:
you'll need to do the toggle here also other the "current" statement will show zero

Reply:
👍

Reply:
Now updated